File: pset6/mario/mario.py
# Program that prints a left-alligned pyramid of hashes according to the user's input
from cs50 import get_int


def main():
    h = get_height()

    for i in range(h):
        for j in range(h):
            if i + j < h - 1:
                print(" ", end="")
            else:
                print("#", end="")
        print()


def get_height():
    while True:
        n = get_int("number 1-8: ")
        if n >= 1 and n <= 8:
            break
    return n


main()

File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //Prompt user for input
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);

    //Print # according to users input
    for (int i = 0; i < n; i++)
    {
        for (int j = 0; j < n; j++)
        {
            if (i + j < n - 1)
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }
        printf("\n");
    }

//End of Code
}